esmvaltool/utils/recipe_test_workflow/doc/source/user_guide/adding_a_recipe.rst

```diff
@@ -0,0 +1,97 @@
+Adding a recipe to the Recipe Test Workflow (|RTW|)
+===================================================
+
+.. include:: ../common.txt
+
+**Please note:** Before you follow these steps to add your recipe, you must be
+able to successfully run it with ESMValtool without error.
+
+* Run ESMValTool locally with your recipe, make sure to take note of the memory
+  and time expenditure provided by the terminal output::
+
+    esmvaltool run <your_recipe_name.yml>
+
+* Run the same recipe on JASMIN, again taking note of the memory and time
+  expenditure printed to the terminal.
+
+* Stop any running workflows::
+
+    cylc stop "*"
+
+* Add the recipe to the ``[task parameters]`` section of the |RTW| ``flow.cylc``
+  workflow file (make sure to place it within the "fast" or "medium" category
+  depending on how long it took to run):
+
+* Run the workflow (it should fail)::
+
+    cylc vip -O <your_site_name>
+
+* Locate the workflow run folder of the workflow you just completed.
+
+* Copy the output files from the "cycle" folder (run/share/bin/cycle) of the
+  workflow you just ran, to your site specific KGO rootpath folder
+  (/data/users/esmval/KGO)::
```

Comment:
KGO rootpath is machine-specific, so please refer to the file/variable where it is set, rather than giving this site-specific example.

Reply:
I've had a go at a change for this in commit 06f79f23d